src/python/grpcio/grpc/extensions/__init__.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2015-2016 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC Python extension API."""
+
+"""The client config of the gRPC extensions module.
+Its value is a proto defined by client_config.proto"""
+CHANNEL_ARG_GRPC_EXTENSIONS_CLIENT_CONFIG = "grpc.extensions.client_config"
+
```

Comment:
I think we need have extension-specific config proto. 

In this case, it would be something like "grpc.extensions.channel_management".

Also, I think affinity based channel management should have its own config proto (which will contain the basic channel_management config as a field). E.g. as "grpc.extensions.affinity_channel_management"

Reply:
And config protos need go to the grpc tree (i.e. language-agnostic).

Reply:
I remove the client_config.proto from this PR, thus we can focus on the bootstrapping API itself.